__tests__/app.test.js

```diff
@@ -418,3 +418,128 @@ describe('GET /api/reviews', () => {
     });
  })
 
+ describe('GET /api/reviews (queries)', () => {
+    test('200: Allows for sorting by a given category via a query', () =>{ 
+        return request(app)
+        .get('/api/reviews?category=dexterity')
+        .expect(200)
+        .then(({body}) => {
+            const reviews = body.reviews;
```

Comment:
The length of the 'reviews' array must be checked to avoid false positive tests.